db/schema.rb

```diff
@@ -619,6 +621,10 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.jsonb "original_text"
+    t.boolean "has_cover_letter", default: false, null: false
+    t.string "cover_letter_logo_type", default: "report_logo", null: false
+    t.string "cover_letter_description"
+    t.string "cover_letter_sender"
```

Comment:
I don't see migration for this

Reply:
I must have put it by mistake from a different branch, I will just delete it as it's not related to this task